### PR TITLE
#164069297 upgrade django 1.0 to django 2.1

### PR DESCRIPTION
**Pull request by akram256**
**What does this PR do?**
* Upgrades codebase from Django 1.10 to Django 2.1

**Description of the tasks to be completed?**
* Update requirements using 
* Refactor URLs to a format that suites Django 2.1 

**How should this be manually tested?**

* Install project requirements using `pip install -r requirements.txt`
* Check for Django version

**What are the relevant pivot tracker stores?**
#164069297

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -91,3 +91,6 @@ ENV/
 
 # SQLite3
 db.sqlite3
+# iDE
```
Comment (Stanley-Okwii) [inline .gitignore]: @akram256 , Could you please rename this to `IDE`. Otherwise nice work

Comment (akram256) [reply .gitignore]: Hey @Stanley-Okwii , i have worked on it. Thank you so much

**Comment by nyabongoedgar** [comment]
Well done.

**Comment by NLSanyu** [comment]
The updated Django version is working perfectly. Well done.

**Comment by PatrickCmd** [comment]
@akram256 Can you please change all the links in the settings file to point to Django 2.10 instead of 1.10 i.e **https://docs.djangoproject.com/en/1.11/topics/settings/** should be **https://docs.djangoproject.com/en/2.1/topics/settings/**

**Comment by PatrickCmd** [comment]
@akram256 Can also change the base branch of this PR to develop branch instead of master

**Pull request merged**